build/configs/artik05x/artik05x_download.sh

```diff
@@ -182,7 +182,7 @@ download()
 
 	# Download all binaries using openocd script
 	pushd ${OPENOCD_DIR_PATH} > /dev/null
-	${OPENOCD} -f ${CFG_FILE} -s ${SCRIPTS_PATH} -c "${commands} exit" || exit 1
+	${OPENOCD} -f ${CFG_FILE} -s ${SCRIPTS_PATH} -c "${commands}" -c "init; $strcmd reset; exit" || exit 1
```

Comment:
What is ```$strcmd```?

Reply:
`strcmd` is a list of commands to be executed before reset. There is no such commands as of now so I have removed it.